perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/impl/MailManagerImpl.java

```diff
@@ -2115,4 +2141,44 @@ private MimeMessage getInvitationMessage(Vo vo, Group group, String language, St
 		return message;
 	}
 
+	/**
+	 * Get notification emails for given VO. If toEmail is not specified, all VO admins are used.
```

Comment:
It is better to write full JavaDoc with parameters, return values, and so on, when when it is only a private method.